api-controllers/ServiceFabrikApiController.js

```diff
@@ -851,14 +899,15 @@ class ServiceFabrikApiController extends FabrikBaseController {
   }
 
   getUpdateSchedule(req, res) {
-    req.manager.isAutoUpdatePossible();
```

Comment:
Here how are we ensuring that for docker plans it fails?

Reply:
Discussed and detailed in this issue.
@kamath-prasad @subhankarc @amitmalav

Reply:
Awesome.